client/packages/electron/src/electron.ts

```diff
@@ -161,7 +162,7 @@ const start = (): void => {
       protocol,
       ip,
       clientVersion: clientVersion || '',
-      isLocal: ip === thisIp.address() || ip === '127.0.0.1',
+      isLocal: ip === address() || ip === '127.0.0.1',
```

Comment:
I think it's good to add a comment or rename address when importing (just to know that address() actually mean current machine ip address)

Reply:
I had thought the same, but didn't implement.. done now in ccd825a0e